pkg/v3/release.go

```diff
@@ -100,3 +111,15 @@ func getLatestRelease(releaseName string, cfg *action.Configuration) (*release.R
 func getReleaseVersionName(rel *release.Release) string {
 	return fmt.Sprintf("%s.v%d", rel.Name, rel.Version)
 }
+
+func getKubernetesServerVersion(kubeConfig common.KubeConfig) (string, error) {
```

Comment:
Moved from common.go

Reply:
It doesn't need to be 😄. There was only one call to it and it was in the same file. When I moved the call to it, I would have either had to export the function, or move the whole function. I decided to move it, but I can move it back. Whatever works